examples/channel-buffering/channel-buffering.go

```diff
@@ -1,9 +1,9 @@
-// By default channels are _unbuffered_, meaning that they
-// will only accept sends (`chan <-`) if there is a
-// corresponding receive (`<- chan`) ready to receive the
-// sent value. _Buffered channels_ accept a limited
-// number of  values without a corresponding receiver for
-// those values.
+// Per default i channels vengono creati senza _buffer_. 
+// Scrivere un valore in un channel (`chan <-`) sarà quindi bloccante
+// non c'è nessuno in ascolto su quel channel (`<- chan`).
```

Comment:
nessuno sembra quasi ci debba essere una persona in ascolto. Non è meglio `nessun ricevitore in ascolto`?